### PR TITLE
media-plugins/vdr-satip: version bump

### PR DESCRIPTION
**Pull request by madmartin**
new upstream version 2.2.5
add proxy maintainership

Closes: https://bugs.gentoo.org/667946
Package-Manager: Portage-2.3.49, Repoman-2.3.10

### CODE REVIEW

File: media-plugins/vdr-satip/vdr-satip-2.2.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+DESCRIPTION="VDR Plugin: integrates SAT>IP network devices seamlessly into VDR"
+HOMEPAGE="http://www.saunalahti.fi/~rahrenbe/vdr/satip/"
+SRC_URI="http://www.saunalahti.fi/~rahrenbe/vdr/satip/files/${P}.tgz"
+
+KEYWORDS="~amd64 ~arm ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+IUSE=""
+
+DEPEND=">=media-video/vdr-2.2.0
```
Comment (Zlogene) [inline media-plugins/vdr-satip/vdr-satip-2.2.5.ebuild]: fix your tabs please

Comment (madmartin) [reply media-plugins/vdr-satip/vdr-satip-2.2.5.ebuild]: @Zlogene 
do you mean tabs like this: 
```
RDEPEND="
    sys-fs/btrfs-progs
    virtual/cron
    systemd? ( sys-apps/systemd )
"

?

**Comment by madmartin** [comment]
@Zlogene please review again, I hope the DEPENDS indentation meets your expectations now

**Comment by juippis** [comment]
Yep, seems like the package could use some caretaking. Please rebase against fresh master and update the copyright header on your files.

**Comment by madmartin** [comment]
> Yep, seems like the package could use some caretaking. Please rebase against fresh master and update the copyright header on your files.

@juippis @Zlogene please review and decide to accept or reject this PR. I added the changes you requested, now its your time to act.

**Comment by juippis** [comment]
Looks good. Please fix the following QA error: 
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/lib64/vdr/plugins/libvdr-satip.so.2.2.0
```
Invoked by settings used: 
https://wiki.gentoo.org/wiki/Package_testing#make.conf

Also please squash your final state into one commit, when you're ready.

**Comment by madmartin** [comment]
>  * QA Notice: Files built without respecting CFLAGS have been detected
>  *  Please include the following list of files in your report:
>  * /usr/lib64/vdr/plugins/libvdr-satip.so.2.2.0

Yes, this is true, and this is correct by design.
As you can see, the result of this package is a so-file, used as a plugin for the main executable /usr/bin/vdr from media-video/vdr . So the Makefile of a vdr plugin obtains the exact CFLAGS which were used to compile /usr/bin/vdr with the statement

`pkg-config --variable=cflags /usr/lib64/pkgconfig/vdr.pc`

to avoid problems.

Look into Makefile to see more details.

**Comment by juippis** [comment]
But you need to show in the ebuild that you are aware of it, so you won't receive bug reports about the subject. So add the following below *DEPEND:
`QA_FLAGS_IGNORED="/usr/lib64/vdr/plugins/libvdr-satip.so.2.2.0"`

Well, to also match 32-bit systems, either
```
/usr/lib(64)?/vdr/plugins/libvdr-satip.so.2.2.0
/usr/lib*/vdr/plugins/libvdr-satip.so.2.2.0
/usr/lib64/vdr/plugins/libvdr-satip.so.2.2.0 /usr/lib/vdr/plugins/libvdr-satip.so.2.2.0
```
should work. You can choose. Also please test with the settings I gave above to properly check it's ignored now.

**Comment by madmartin** [comment]
Very interesting, I wasn't aware of that before.

I did a lot of testing and chose 

`QA_FLAGS_IGNORED="usr/lib/vdr/plugins/libvdr-satip.* usr/lib64/vdr/plugins/libvdr-satip.*"`

Other than stated in https://devmanual.gentoo.org/eclass-reference/ebuild/index.html it looks to me that regular expressions in the middle of the file names do not work. 
Side note: the trailing digits are equal to the version of media-video/vdr against which a vdr plugin is compiled.
I will add QA_FLAGS_IGNORED to all the other vdr plugins in the future.

**Comment by juippis** [comment]
Thanks!

**Pull request closed without merging**